README.md

```diff
@@ -1,5 +1,11 @@
 # CONTRAST RATIO APP
 
+## Prerequisites
+
+Install Node.js in order to have access to npm.
```

Comment:
I think wrapping the link in a text will look better, I believe this is the markdown syntax `[text](url)`, you can convert the install into a link our create a new sentence like `click to install node.js .

Reply:
Cool. You know you can actually request changes via GitHub interface? Remember I did that reviewing your PR?